bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZZTo4l0To3jets/ZZNJets_Zll_VBF_LO_proc_card.dat

```diff
@@ -0,0 +1,14 @@
+import model sm-ckm_no_b_mass
+
+define p = u c s d b u~ c~ s~ d~ b~ g
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+define j = p
+# Specify process(es) to run
+generate p p > z z @1
+add process p p > z z j @2
+add process p p > z z j j @3
+add process p p > z z j j j @4
+#add process p p > z z j j j j @5
+# Output processes to MadEvent directory
+output ZZNJets_Zll_VBF_LO  -nojpeg
```

Comment:
Please remove "VBF" in the output name, this could be misleading

Reply:
hi @sihyunjeon, these cards are used in gridpacks for the Higgs to ZZ VBF differential XS analysis. Indeed, the VBF topology assumes ZZ+2jets only. Is this the only reason to remove the VBF from the name? (in this case gridpacks have to be reproduced only because of the name)

Reply:
The gridpacks already have some settings that doesn't really seem OK for me (pdfwgt, lhaid, ...). So I would still ask you to change the name. The gridpack itself does not include VBF physics process, whether it is used for VBF H to ZZ analysis or not. I believe gridpack/sample info should be clear to all CMS people.